.postcssrc.yml

```diff
@@ -22,20 +18,6 @@ plugins:
       - 'last 2 versions'
       - '> 1%'
 
-    # Instruct all plugins to omit pre-polyfilled CSS
-    # https://github.com/csstools/postcss-preset-env#preserve
-    preserve: false
```

Comment:
If we won't replace css variables, we need to move them from :root {} to the editor scope to prevent confllicts